.github/workflows/docker-publish.yml

```diff
@@ -24,7 +24,9 @@ jobs:
         run: docker build -t discovery_ui:build .
```

Comment:
Do you need to inject the secrets into the environment here?

Reply:
*facepalm* Yes...just did that.